query.go

```diff
@@ -619,23 +745,20 @@ func (s *ChainService) prepareCFiltersQuery(blockHash chainhash.Hash,
 	}
 	bestHeight := int64(bestBlock.Height)
 
-	qo := defaultQueryOptions()
```

Comment:
I think rather than remove the old set of options, we should instead seek to unify them. If we need to add extra options that are relevant to the main package, but not the `query` package, then we can use a type alias here.

Reply:
> If we need to add extra options that are relevant to the main package, but not the query package, then we can use a type alias here.

can you elaborate on "we can use a type alias here"?